sprint9-js-objects/ex1.js

```diff
@@ -0,0 +1,14 @@
+function madePizza(titlu, portii, ingrediente) {
```

Comment:
```suggestion
function getReceipt(titlu, portii, ingrediente) {
```

optional - nice - to have - denumiri mai sugestive, abordarbile in toate situatiile